tkupchyn/homework_03/next_happy_year.py

```diff
@@ -54,8 +54,8 @@ def next_happy_year_alternative(year: int) -> int:
     if year not in range(1000, 9001):
         raise ValueError(f"Year {year} is out of range")
 
-
-    year += 1 if is_happy_year(year) else year
```

Comment:
@kudinov-fedor Could you please look into this and tell me what I've done wrong here

Reply:
this is your logic:

year = year + 1 if is_happy_year(year) else year + year

Reply:
you need 

year = year + 1 if is_happy_year(year) else year

Reply:
Thanks, now I've got the difference